### PR TITLE
Feature/updated swagger

### PR DESCRIPTION
**Pull request by kevinkirkup**

### CODE REVIEW

File: docs/client/AccountConsentApi.md [hunk 1]
```diff
@@ -31,23 +33,27 @@ Name | Type | Description  | Notes
 
 ### HTTP request headers
 
- - **Content-Type**: application/json
- - **Accept**: application/json
+- **Content-Type**: Not defined
```
Comment (Try75) [inline docs/client/AccountConsentApi.md]: Is this change correct?

File: docs/client/AccountInvitationsApi.md [hunk 2]
```diff
@@ -117,32 +131,38 @@ Name | Type | Description  | Notes
...
 ### Optional Parameters
+
 Optional parameters are passed through a pointer to a InviteAccountOpts struct
 
+
 Name | Type | Description  | Notes
 ------------- | ------------- | ------------- | -------------
 
- **body** | [**optional.Interface of AccountInvite**](AccountInvite.md)|  | 
+ **accountInvite** | [**optional.Interface of AccountInvite**](AccountInvite.md)|  |
```
Comment (Try75) [inline docs/client/AccountInvitationsApi.md]: body -> accountInvite seems like a fix, but just want to make sure it is intentional.

File: docs/client/AccountInvite.md [hunk 3]
```diff
@@ -1,15 +1,16 @@
 # AccountInvite
 
 ## Properties
+
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**Account** | [***Link**](Link.md) |  | 
+**Account** | [**Link**](Link.md) |  |
```
Comment (Try75) [inline docs/client/AccountInvite.md]: I've noticed this removal of one "*" from a number of Links. Is this a fix also?

File: docs/client/ConnectionsApi.md [hunk 4]
```diff
@@ -195,60 +223,134 @@ Name | Type | Description  | Notes
...
+- **Content-Type**: Not defined
+- **Accept**: application/json
 
-[[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints) [[Back to Model list]](../README.md#documentation-for-models) [[Back to README]](../README.md)
+[[Back to top]](#) [[Back to API list]](../README.md#documentation-for-api-endpoints)
+[[Back to Model list]](../README.md#documentation-for-models)
+[[Back to README]](../README.md)
+
+
+## RespawnConnection
```
Comment (Try75) [inline docs/client/ConnectionsApi.md]: This isn't an admin API call that should not be exposed to the end user, is it?

File: docs/client/GatewaysApi.md [hunk 5]
```diff
@@ -4,34 +4,115 @@ All URIs are relative to *https://api.pureport.com*
...
+[**CreateGatewayTask**](GatewaysApi.md#CreateGatewayTask) | **Post** /gateways/{gatewayId}/tasks | Add a new task
 [**GetGatewayBGPRoutes**](GatewaysApi.md#GetGatewayBGPRoutes) | **Get** /gateways/{gatewayId}/bgpRoutes | Get gateway bgp routes
+[**GetGatewayTasks**](GatewaysApi.md#GetGatewayTasks) | **Get** /gateways/{gatewayId}/tasks | List connection tasks
 
 
-# **GetGatewayBGPRoutes**
+
+## CreateGatewayTask
+
+> Task CreateGatewayTask(ctx, gatewayId, optional)
```
Comment (Try75) [inline docs/client/GatewaysApi.md]: Are Tasks something that get exposed to users?

**Comment by kevinkirkup** [comment]
> Since this is already published, let's not be concerned with changes here, but we will with the next release.

I've added reminders to check these in the user story: PE-1929.

**Pull request merged**